src/components/Layout/TopNav.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Navbar, Nav } from 'react-bootstrap';
+import Styles from "../../styles/nav.module.scss"
+
+export const LoaderLine = () => (
+  <div>
+    <span className={Styles.expand} />
+  </div>
+)
+
+const TopNav = (props) => {
+  return (
+    <div>
+      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar.Brand href="#home">I<span className={Styles.ibookBrand}>book</span>ing</Navbar.Brand>
+        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+        <Navbar.Collapse id="responsive-navbar-nav">
+          <Nav className="mr-auto">
+            <Nav.Link href="/Tasker">Tasker</Nav.Link>
```

Comment:
Could we change `Task` and `Tasker` here? As we have different access permission, the tasker should not access `tasker` in nav. May be changed in other content, such as service, guide.

Reply:
done